app/src/userpages/modules/canvas/actions.js

```diff
@@ -106,31 +106,36 @@ export const getCanvases = () => (dispatch: Function, getState: () => StoreState
 
 export const getCanvas = (id: CanvasId) => (dispatch: Function) => {
     dispatch(getCanvasRequest(id))
-    return get(`${apiUrl}/${id}`)
+    return api.get(`${apiUrl}/${id}`)
         .then(handleEntities(canvasSchema, dispatch))
         .then((data) => dispatch(getCanvasSuccess(data)))
         .catch((e) => {
             dispatch(getCanvasFailure(e))
-            dispatch(errorNotification({
-                title: 'Error!',
-                message: e.message,
-            }))
+            Notification.push({
+                title: e.message,
+                icon: NotificationIcon.ERROR,
+            })
             throw e
         })
 }
 
-export const deleteCanvas = (id: CanvasId) => (dispatch: Function) => {
+export const deleteCanvas = (id: CanvasId) => async (dispatch: Function): Promise<void> => {
     dispatch(deleteCanvasRequest(id))
-    return del(`${apiUrl}/${id}`)
-        .then(() => dispatch(deleteCanvasSuccess(id)))
-        .catch((e) => {
-            dispatch(deleteCanvasFailure(e))
-            dispatch(errorNotification({
-                title: 'Error!',
-                message: e.message,
-            }))
-            throw e
+    try {
+        const deleteCanvas = await api.del(`${apiUrl}/${id}`)
+        dispatch(deleteCanvasSuccess(id))
+        Notification.push({
+            title: 'Canvas deleted successfully',
```

Comment:
Let's add these to the translation file. I know the editor is not translated but everything in the user pages has been translated so far :)